lifterlms-groups/automatically-create-group-with-group-admin-when-purchased-via-woocommerce.php

```diff
@@ -0,0 +1,61 @@
+<?php // Do not copy this line!
+/**
+ * Create a LifterLMS Group for every WooCommerce order. Set the number of seats
+ * to be equal to the "quantity" of items bought in WooCommerce.
+ *
+ * You can add this recipe to your site by creating a custom plugin
+ * or using the Code Snippets plugin available for free in the WordPress repository.
+ * Read this companion documentation for step-by-step directions on either method.
+ * https://lifterlms.com/docs/adding-custom-code/
+ *
+ * @param int $order_id The WooCommerce order ID.
+ *
+ * @return void
+ */
+function create_llms_group_with_admin($order_id)
+{
+    if (class_exists('LLMS_Group') && class_exists('LLMS_Groups_Enrollment') ) {
```

Comment:
It's not critical, but possible to add spacing before/after brackets here and elsewhere? Thanks!